projects.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: page
+title: Projects
+parent: Get Involved
+order: 1
+---
+
+Please ask in #welcome:tomesh.net if you would like to learn more or contribute to any one of these projects.
+
+## Nodes/Network
+
+### Raspberry/Orange Pi Install Script + Bi-Weekly Virtual Mesh
+Description: Installation script for Raspbery and Orange Pi devices from a stock firmware.   
+Status: See [Github Issues](https://github.com/tomeshnet/prototype-cjdns-pi/issues)  
+Contact: benhylau  
+[More Info](https://github.com/tomeshnet/prototype-cjdns-pi2)
+
+
+### Mesh Orange
```

Comment:
I think we want more discriptive names, like "Build low-cost mesh nodes using a mesh orange"